Makefile

```diff
@@ -23,37 +25,37 @@ setup: .runner .features .submitter .datastore .filestore .pdf-generator .servic
 destroy: stop clean
 
 make-features:
-	mkdir -p .features/email-output
-	mkdir -p .features/json-output
-	mkdir -p .features/save-and-return-module
+	./scripts/make_features.sh
 
 copy-features:
-	cp -r .runner/* .features/email-output
-	cp -r .runner/* .features/json-output
-	cp -r .runner/* .features/save-and-return-module
+	./scripts/copy_features.sh
+
+make-components:
+	./scripts/make_components.sh
+
+copy-components:
+	./scripts/copy_components.sh
 
 stop:
 	docker-compose down
 
 build: stop setup
-	echo HEAD > .features/email-output/APP_SHA
-	mkdir -p .features/email-output/form
-	cp -r ./forms/features/email-output/* .features/email-output/form
-	echo HEAD > .features/json-output/APP_SHA
-	mkdir -p .features/json-output/form
-	cp -r ./forms/features/json-output/* .features/json-output/form
-	echo HEAD > .features/save-and-return-module/APP_SHA
-	mkdir -p .features/save-and-return-module/form
-	cp -r ./forms/features/save-and-return-module/* .features/save-and-return-module/form
+	./scripts/setup_features.sh
+	./scripts/setup_components.sh
 	docker-compose build --parallel
 
 serve: build
 	docker-compose up -d
+	./scripts/wait_for_datastore_app.sh
+	./scripts/wait_for_filestore_app.sh
+	./scripts/wait_for_submitter_app.sh
+	./scripts/wait_for_service_token_cache_app.sh
 	./scripts/wait_for_features_apps.sh
+	./scripts/wait_for_components_apps.sh
```

Comment:
Was outputting the SHA's a useful thing?

Reply:
That's a good question. It was implemented by Emile back in October/November

All it prints is `{"APP_SHA":"HEAD"}` so ... suggestions? I mean, it _confirms_ that the response is as expected, but thassit

(Would it error if grep doesn't find that ..?)